MemorialHouse/MHPresentation/MHPresentation/Source/Audio/AudioViewController.swift

```diff
@@ -0,0 +1,353 @@
+        
+        audioRecorder = try? AVAudioRecorder(url: audioFileURL, settings: audioRecordersettings)
+        audioRecorder?.isMeteringEnabled = true
+    }
+    
+    private func configureAddSubviews() {
+        stackView = UIStackView(arrangedSubviews: [cancelButton, titleLabel, saveButton])
+        view.addSubview(stackView)
+        view.addSubview(meteringBackgroundView)
+        view.addSubview(audioButtonBackground)
+        audioButtonBackground.addSubview(audioButton)
+        view.addSubview(timeTextLabel)
+    }
+    
+    private func configureConstraints() {
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        cancelButton.setWidth(60)
+        saveButton.setWidth(60)
+        
+        stackView.setCenterX(view: view)
+        stackView.setAnchor(
+            top: view.topAnchor, constantTop: -10,
+            leading: view.leadingAnchor, constantLeading: 25,
+            trailing: view.trailingAnchor, constantTrailing: 25,
+            height: 120
+        )
+        
+        meteringBackgroundView.backgroundColor = UIColor.mhPink
+        meteringBackgroundView.translatesAutoresizingMaskIntoConstraints = false
+        meteringBackgroundView.setCenterX(view: view)
+        meteringBackgroundView.setAnchor(top: titleLabel.bottomAnchor, constantTop: 24)
+        meteringBackgroundView.setWidthAndHeight(width: 320, height: volumeHalfHeight * 2)
+        meteringBackgroundView.layer.cornerRadius = 25
+        
+        upMeteringView.translatesAutoresizingMaskIntoConstraints = false
+        meteringBackgroundView.addSubview(upMeteringView)
+        upMeteringView.setCenter(view: meteringBackgroundView, offset: CGPoint(x: 0, y: -volumeHalfHeight/2))
+        upMeteringView.setWidthAndHeight(width: 300, height: volumeHalfHeight)
+        
+        downMeteringView.translatesAutoresizingMaskIntoConstraints = false
+        meteringBackgroundView.addSubview(downMeteringView)
+        downMeteringView.setCenter(view: meteringBackgroundView, offset: CGPoint(x: 0, y: volumeHalfHeight/2))
+        downMeteringView.setWidthAndHeight(width: 300, height: volumeHalfHeight)
+        
+        audioButtonBackground.setAnchor(top: meteringBackgroundView.bottomAnchor, constantTop: 20)
+        audioButtonBackground.setCenterX(view: view)
+        audioButtonBackground.setWidthAndHeight(width: 60, height: 60)
+        
+        audioButton.layer.cornerRadius = 24
+        audioButton.translatesAutoresizingMaskIntoConstraints = false
+        audioButtonConstraints = [
+            audioButton.widthAnchor.constraint(equalToConstant: 48),
+            audioButton.heightAnchor.constraint(equalToConstant: 48),
+            audioButton.centerXAnchor.constraint(equalTo: audioButtonBackground.centerXAnchor),
+            audioButton.centerYAnchor.constraint(equalTo: audioButtonBackground.centerYAnchor)
+        ]
+        NSLayoutConstraint.activate(audioButtonConstraints)
```

Comment:
P3: 혹시 이 부분만 저희 Anchor 메서드를 사용하지 않은 이유가 있으실까요??

그리고 아래와 같이 배열을 `.activate()` 내부에 바로 넣어줘도 될 것 같습니당 ~

```suggestion
    audioButton.translatesAutoresizingMaskIntoConstraints = false
    NSLayoutConstraint.activate([
        audioButton.widthAnchor.constraint(equalToConstant: 48),
        audioButton.heightAnchor.constraint(equalToConstant: 48),
        audioButton.centerXAnchor.constraint(equalTo: audioButtonBackground.centerXAnchor),
        audioButton.centerYAnchor.constraint(equalTo: audioButtonBackground.centerYAnchor)
    ])
```

Reply:
수정하겠습니다 !!